Makefile

```diff
@@ -3,20 +3,20 @@ test:
 testcov:
 	py.test --cov=ideascube/ --cov-report=term-missing
 develop:
-	pip install -r requirements-dev.txt
```

Comment:
Sorry for the lag, but as soon as your venv is made pointing to python3 you should not need that changes.
What was the rationale behind this?